<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListaEnc</class>
 <widget class="QDialog" name="ListaEnc">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lista Encadeada</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>489</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #00401A;</string>
    </property>
    <property name="text">
     <string>Valor a ser inserido na lista:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_value">
    <property name="styleSheet">
     <string notr="true">border: 1px solid #B1D34B;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>264</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00401A;</string>
   </property>
   <property name="text">
    <string>Remover valor na posição:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_remove">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>219</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B1D34B;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar valor na lista</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00401A;</string>
   </property>
   <property name="text">
    <string>Posição em que sera inserido:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_pos">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>219</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B1D34B;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_consult">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>219</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #B1D34B;
border-radius: 5px;
padding: 0 8px;
outline: none;
color: #525252;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>264</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #00401A;</string>
   </property>
   <property name="text">
    <string>Consultar posição da lista:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Remover valor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar valor</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStack">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #B1D34B;
border-radius: 8px;
padding: 0 8px;
outline: none;
color: #525252;background: #ffffff
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>